front/public/locales/en/timesStops.json

```diff
@@ -1,18 +1,19 @@
 {
-  "arrivalTime": "Requested arrival Time",
-  "calculatedArrivalTime": "Calculated arrival time",
-  "calculatedDepartureTime": "Calculated departure time",
+  "arrivalTime": "requested arrival time",
+  "calculatedArrivalTime": "calculated arrival time",
+  "calculatedDepartureTime": "calculated departure time",
   "dayCounter": "D+{{count}}",
-  "departureTime": "Requested departure Time",
-  "diffMargins": "Margins diff.",
-  "name": "Name",
+  "departureTime": "requested departure time",
+  "diffMargins": "margins diff. (s)",
+  "name": "CI",
+  "ch": "ch",
   "noPathLoaded": "No path loaded",
-  "realMargin": "Real margin",
-  "receptionOnClosedSignal": "Reception on Closed Signal",
+  "realMargin": "real margin (s)",
```

Comment:
`"realMargin": "real margin (s.)",`

Reply:
I disagree with this. The SI way to write units like seconds does not take a dot, I don't see why we should differ on this.

Reply:
in the mockup there is a dot : 
<img width="1133" alt="image" src="https://github.com/user-attachments/assets/306bd415-100b-4f1d-9a4a-1845f72df32a">

Reply:
This is true, but I believe we should follow the standard unit conventions

Reply:
Perhaps we can ask @thibautsailly for confirmation

Reply:
I did, but in the meantime I think we should defer to the BIPM specifications for units :
> Les symboles d’unités sont des entités mathématiques et non des abréviations. Par conséquent, ils ne doivent pas être suivis d’un point, sauf s’ils se trouvent placés à la fin d’une phrase

Reply:
Thibaut confirmed the points were not intentional